packages/manager/test/mockSlice.test.ts

```diff
@@ -0,0 +1,400 @@
+import { describe, test, expect, vi } from "vitest";
+import { WidgetTypes } from "@prismicio/types-internal/lib/customtypes/widgets";
+import {
+	SharedSlice,
+	SlicesTypes,
+} from "@prismicio/types-internal/lib/customtypes/widgets/slices";
+
+import { isRight } from "fp-ts/lib/Either";
+
+import { GeoPointContent } from "@prismicio/types-internal/lib/documents/widgets/nestable";
+import { LinkContent } from "@prismicio/types-internal/lib/documents/widgets/nestable/Link";
+import { SharedSliceContent } from "@prismicio/types-internal/lib/content";
+import {
+	DiffOperation,
+	SliceDiff,
+} from "@prismicio/types-internal/lib/customtypes/diff";
+import { mockSlice } from "../src/lib/mockSlice";
+
+vi.mock("lorem-ipsum", () => {
+	return {
+		__esModule: true,
+		LoremIpsum: vi.fn().mockImplementation(() => {
+			return {
+				generateParagraphs: vi.fn().mockReturnValue("Some text."),
+			};
+		}),
+	};
+});
+
+vi.mock("@prismicio/mocks/lib/generators/utils/slug", () => {
+	return vi.fn().mockReturnValue("Woo");
+});
+
+describe.skip("mockSlice", () => {
```

Comment:
Is there still a reason to skip these tests? (not sure if it was something from React blocking before)